src/parsers/withdrawals.ts

```diff
@@ -27,15 +27,14 @@ function parseCoreWithdrawalEvent<T>(ev: T): DepoWithdrawEvent {
 }
 
 function parseGRouterWithdrawEvent<T>(ev: T): DepoWithdrawEvent {
-    let usdAmount = ev.params.calcAmount;
+    const shareAmount = ev.params.calcAmount;
```

Comment:
`calcAmount` wasn't supposed to be the usd value of the withdrawal?
Then, what's the difference between fields `tokenAmount` and `calcAmount`?

Reply:
In grouter's withdraw event, the tokenAmount is passed Gtoken's amount and the calcAmount is the withdrawal coin amount.